tools/build_config/build.ps1

```diff
@@ -0,0 +1,139 @@
+param (
+  [int]$vs_version = 2017,
+  [string]$build_config = 'Release',
+  [string]$build_dir = "",
+  [string]$build_tests = "On",
+  [string]$cmake_flags = "",
+  [string]$build_fortran = "OFF",
+
+  [switch]$build,
+  [switch]$test,
+  [switch]$package,
+  [switch]$print_versions
+)
+
+. $PSScriptRoot/powershell_helpers.ps1
```

Comment:
a comment would be helpful explaining which functions you're importing from here